hudi-client/src/test/java/org/apache/hudi/table/action/commit/TestUpsertPartitioner.java

```diff
@@ -269,8 +267,8 @@ public void testUpsertPartitionerWithSmallInsertHandling() throws Exception {
     assertEquals(BucketType.INSERT, partitioner.getBucketInfo(3).bucketType,
         "Bucket 3 is INSERT");
     assertEquals(4, insertBuckets.size(), "Total of 4 insert buckets");
-    assertEquals(0, insertBuckets.get(0).bucketNumber, "First insert bucket must be same as update bucket");
-    assertEquals(200.0 / 2400, insertBuckets.get(0).weight, 0.01, "First insert bucket should have weight 0.5");
+    assertEquals(0, insertBuckets.get(0).getKey().bucketNumber, "First insert bucket must be same as update bucket");
+    assertEquals(200.0 / 2400, insertBuckets.get(0).getKey().weight, 0.01, "First insert bucket should have weight 0.5");
```

Comment:
typo. weight is 0.08

Reply:
just curious, why not asserting every buckets weight?

Reply:
Yes, it should be 0.08, the testcase is not add by me, I am also curious.

Reply:
it is fine. Would you mind adding assertions for all buckets.

Reply:
Sure, I will add for all buckets.